### PR TITLE
Specify guest room access

### PR DESCRIPTION
**Pull request by illicitonion**

### CODE REVIEW

File: scripts/continuserv/main.go [hunk 1]
```diff
@@ -113,6 +113,7 @@ func filter(e fsnotify.Event) bool {
 func serve(w http.ResponseWriter, req *http.Request) {
 	wg.Wait()
 	b := toServe.Load().([]byte)
+	w.Header().Set("Content-Type", "text/html")
```
Comment (Kegsay) [inline scripts/continuserv/main.go]: Why you branch off other PRs? :cry:

File: specification/modules/guest_access.rst [hunk 2]
```diff
@@ -0,0 +1,78 @@
+Guest access
+============
+
+.. _module:guest-access:
+
+There are times when it is desirable for clients to be able to interact with
+rooms without having to fully register for an account on a homeserver or join
+the room. This module specifies how these clients should interact with servers,
```
Comment (Kegsay) [inline specification/modules/guest_access.rst]: , on this line can be removed.

File: specification/modules/guest_access.rst [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+present with ``history_visibility`` value ``world_readable``. These events may
+be returned even if the anonymous user is not joined to the room.
+
+Servers MUST only allow guest users to join rooms if the ``m.room.guest_access``
+state event is present on the room, and has the ``guest_access`` value
+``can_join``. If the ``m.room.guest_access`` event is changed to stop this from
+being the case, the server MUST set those users' ``m.room.member`` state to
+``leave``.
+
+Security considerations
```
Comment (Kegsay) [inline specification/modules/guest_access.rst]: Missing any information on enforcing captchas on registering guest accounts. We SHOULD be recommending this.

**Comment by Kegsay** [comment]
LGTM other than this.

File: specification/modules/guest_access.rst (hunk 2 above)
Comment (illicitonion) [inline specification/modules/guest_access.rst]: Done

File: specification/modules/guest_access.rst (hunk 3 above)
Comment (illicitonion) [inline specification/modules/guest_access.rst]: Added a comment. I imagine you want something stronger, but I don't. We can fight about it some time.

Comment (Kegsay) [inline specification/modules/guest_access.rst]: Nope, that's all I wanted :smile:  it's up to the individual HS admins to set their registration rules, I just wanted a nudge in this section is all.

**Comment by Kegsay** [comment]
LGTM

**Pull request closed without merging**